dvc/command/machine.py

```diff
@@ -392,7 +408,7 @@ def add_parser(subparsers, parent_parser):
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
     machine_status_parser.add_argument(
-        "name", help="Name of the running machine."
+        "name", nargs="?", help="(optional) Name of the running machine."
```

Comment:
```suggestion
        "name", nargs="?", help="(optional) Name of the machine."
```

I think it's self-explanatory, and the `machine` isn't running, the instance of it is.